src/measurements/ad-blocker.js

```diff
@@ -0,0 +1,49 @@
+import { AddonManager } from 'resource://gre/modules/AddonManager.jsm';
+import BaseMeasurement from '../lib/measurement';
+
+// This list is constructed by manually-reviewing all add-ons in the AMO
```

Comment:
This isn't a bad thought, but for fairness' sake I trying to come up with a deterministic methodology for inclusion. Pragmatically speaking, this already grabs 99%+ of users, which is probably good enough for these purposes.